tests/test_ipc.py

```diff
@@ -118,6 +123,51 @@ def test_IPC():
         ],
     }
 
+    # Deal with Windows weirdness
+    if platform == "win32":
+        expected_output = {
+            "id": 0,
+            "type": "response",
+            "cancelled": False,
+            "command": HIGHLIGHT,
+            "result": [
+                ((1, 0), 4, "Keyword"),
+                ((1, 5), 4, "Name"),
+                ((1, 10), 6, "Keyword"),
+                ((1, 17), 1, "Name"),
+                ((1, 20), 12, "Comment"),
+                ((3, 0), 3, "Name"),
+                ((3, 4), 1, "Operator"),
+                ((3, 6), 3, "Name"),
+                ((3, 11), 7, "Comment"),
+                ((5, 0), 5, "Name"),
+                ((5, 5), 1, "Punctuation"),
+                ((5, 6), 5, "String"),
```

Comment:
Note that the hidden char takes up three character widths instead of one and that the hidden char was not found in the text (https://github.com/Moosems/salve/blob/c7ef978121213f21143a614a9de65434c486bdba/salve_ipc/server_functions/highlight/links_and_hidden_chars.py#L99-L109)